charts/super-agent-deployment/templates/uninstall-job.yaml

```diff
@@ -0,0 +1,47 @@
+{{- if .Values.cleanupManagedResources -}}
+{{- $uninstallJobName := include "newrelic.common.naming.truncateToDNSWithSuffix" ( dict "name" (include "newrelic.common.naming.fullname" .) "suffix" "uninstall-job" ) -}}
+{{- /*
+The resources managed by the super-agent and the label selector are hardcoded on the super-agent.
+*/ -}}
+{{- $saResourceList := (list "configmaps") -}}
+{{- $saCRList := (list "helmreleases.helm.toolkit.fluxcd.io" "helmrepositories.source.toolkit.fluxcd.io") -}}
+{{- $saResourcesLabelSelector := "app.kubernetes.io/managed-by=newrelic-super-agent" -}}
+{{- /*
+To understand why this job installs manifests instead of using Helm, read the comment at `job-manifests.yaml`.
+*/ -}}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+    helm.sh/hook-weight: "10010"
+  name: {{ $uninstallJobName }}
+  namespace: {{ .Release.Namespace }}
+spec:
+  ttlSecondsAfterFinished: 120
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: {{ include "newrelic.common.serviceAccount.name" . }}
+      containers:
+        - name: delete-crs
+          image: bitnami/kubectl
+          command:
+            - bash
+          args:
+            - -c
+            - |
+              set -o pipefail
+              {{- /* Delete the standard resources managed by the super-agent */ -}}
+              {{ range $i, $resourceName := $saResourceList }}
+              kubectl delete {{ $resourceName }} -l {{ $saResourcesLabelSelector }}
```

Comment:
I would restrict the namespace, currently I am not sure what it would do

Reply:
Changed here: [426f796](https://github.com/newrelic/helm-charts/pull/1278/commits/426f79658f207031c8c128f9d3bfded3b70d1b49)